tutorials/arvo/claz.md

```diff
@@ -0,0 +1,76 @@
++++
+title = "claz"
+weight = 4
+template = "doc.html"
++++
+
+This document outlines some advanced techniques for interacting with Azimuth contracts from Urbit itself. It's strongly discouraged to do this unless you're very confident as Azimuth transactions will cost Etherium to complete. However, until you upload a signed transaction to the block chain no transaction will take place so it's safe to explore these tools. It will also be very helpful to have at least a rudimentary understanding of Hoon in order to read and modify parts of the code.
```

Comment:
The primary risk here isn't transaction cost so much as, you know, doing a bad transfer or whatever.

> modify parts of the code

This is exactly why this isn't suited for public use. (^: